questions/functionconditions.md

```diff
@@ -20,19 +24,13 @@ First, set up a dummy scoreboard objective:
 
     scoreboard objectives add success_score dummy
 
-We're going to use the executing entity to store the result. This can of course be substituted for a fake player or a different entity as well.
-
-Make sure the score is initiated (not null). Easy way to do this is by adding 0:
-
-    scoreboard players add @s success_score 0
-
-----
+We're going to use the executing entity to store the result. This can of course be substituted for a [fake player](/wiki/questions/fakeplayer) (because it's about checking a block, independently of an entity, it's recomended using it) or a different entity as well.
```

Comment:
we still have the issue of saying one thing and doing another right below. It looks like we're going with the fake player, so here is the changed text:

```suggestion
We're going to use a [fake player](/wiki/questions/fakeplayer) (because it's about checking a block, independently of an entity, it's recomended to use it) to store the result. This can of course be substituted for the executing entity or a different entity as well.
```